packages/client/components/GitLabScopingSearchCurrentFilters.tsx

```diff
@@ -0,0 +1,86 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import {PALETTE} from '../styles/paletteV3'
+import {GitLabScopingSearchCurrentFilters_meeting$key} from '../__generated__/GitLabScopingSearchCurrentFilters_meeting.graphql'
+
+const Wrapper = styled('div')({
+  width: '100%',
+  display: 'flex',
+  paddingLeft: '72px',
+  paddingTop: '8px'
+})
+
+const Description = styled('div')({
+  color: PALETTE.SLATE_600,
+  fontSize: 16,
+  fontWeight: 500,
+  whiteSpace: 'nowrap'
+})
+
+const Items = styled('div')({
+  color: PALETTE.SLATE_600,
+  fontSize: 16,
+  fontWeight: 600,
+  fontStyle: 'italic',
+  padding: '0px 24px 0px 4px',
+  width: '100%',
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis'
+})
+
+interface Props {
+  meetingRef: GitLabScopingSearchCurrentFilters_meeting$key
+}
+
+const GitLabScopingSearchCurrentFilters = (props: Props) => {
+  const {meetingRef} = props
+  const meeting = useFragment(
+    graphql`
+      fragment GitLabScopingSearchCurrentFilters_meeting on PokerMeeting {
+        gitlabSearchQuery {
+          selectedProjectsIds
+        }
+        viewerMeetingMember {
+          teamMember {
+            integrations {
+              gitlab {
+                projects {
+                  ... on _xGitLabProject {
+                    id
+                    fullPath
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    `,
+    meetingRef
+  )
+  const {gitlabSearchQuery, viewerMeetingMember} = meeting
+  const {selectedProjectsIds} = gitlabSearchQuery
+  const projects = viewerMeetingMember?.teamMember.integrations.gitlab.projects
+
+  const selectedProjectsNames = [] as string[]
```

Comment:
-1 little messy here, can simplify to

```ts
  const selectedProjectsNames = projects
    .filter((project) => selectedProjectIds.contains(project.id))
    .map(({fullPath}) => fullPath)
  const currentFilters = selectedProjectsNames.length === 0 ? 'None' : selectedProjectsNames.join(', ')
```

Reply:
Nice, the `join` method is cleaner. I've updated the three filters to include that.

However, I think we want to loop through `selectedProjectIds` instead of projects so that newly selected project filters are appended to the list. 

For example, if `const projects = ['DEV', 'DESIGN', 'TEST']`, and the text says `Current filters: DESIGN`. If the user then also selects the `DEV` project, I'd expect the text to update to `Current filters: DESIGN, DEV`. 

So, I've kept some of the old `forEach` and `find` logic.